src/Pages/Wishlist/Wishlist.css

```diff
@@ -55,17 +55,25 @@
     color: var(--white);    
 }
 
+.container-wishlist-empty {
+    position: relative;
+}
+
+.img-empty-wishlist {
+    height: 25rem;
+    width: 100vw;
+    object-fit: contain;
+}
+
+.btn-empty-wishlist {
+    position: absolute;
+    top: 50%;
```

Comment:
Try to avoid using larger values in top it will affect in responsiveness later  instead use try to use flex or grid